test/uk/gov/hmrc/tai/connectors/TaxFreeAmountComparisonConnectorSpec.scala

```diff
@@ -44,7 +44,7 @@ class TaxFreeAmountComparisonConnectorSpec extends BaseSpec {
           "data" -> Json.obj(
             "previous" -> Json.arr(
               Json.obj(
-                "componentType" -> CarBenefit,
+                "componentType" -> Json.toJsFieldJsValueWrapper[TaxComponentType](CarBenefit),
```

Comment:
Don't think `toJsFieldJsValueWrapper` is quite right here, this should be:

```suggestion
                "componentType" -> Json.toJson[TaxComponentType](CarBenefit),
```